src/scs/slsa-framework.md

```diff
@@ -0,0 +1,22 @@
+# SLSA Framework
+
+Supply chain Levels for Software Artifacts (SLSA) is a security framework for tampering prevention, integrity improvement and securing packages and infrastructure of a project.
+
+## Levels of assurance
+
+One of the requirements for the solution is to reach SLSA Level 4 and even go beyond that. This requires a lot of process changes as well as technical work. Current status for Thincompute project is visualized at the separate wiki page: Supply Chain Security 
```

Comment:
... the separate wiki page: Supply Chain Security 

I think we should not use internal links here. To what do you want to refer here?